frappe/model/mapper.py

```diff
@@ -193,7 +193,10 @@ def map_fields(source_doc, target_doc, table_map, source_parent):
 	for df in target_doc.meta.get("fields"):
 		if df.fieldname not in no_copy_fields:
 			# map same fields
-			val = source_doc.get(df.fieldname)
+            if df.fieldname[:7] == "custom_":
```

Comment:
This isn't a good idea. 

1. In most cases source and target both will have `custom_` prefix. 
2. If not, you can rename the field: https://github.com/frappe/frappe/pull/22513